db/migrate/20160104131622_remove_deprecated_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDeprecatedColumns < ActiveRecord::Migration
+  def change
+    remove_column :subject_workflow_counts, :set_member_subject_id
```

Comment:
ha - i just read the note. we can choose when to migrate so this will work, sorry should have read it properly.